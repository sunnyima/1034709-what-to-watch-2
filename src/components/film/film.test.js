import React from 'react';
import renderer from 'react-test-renderer';
import Film from "./film";

it(`Film correctly renders after relaunch`, () => {
  const tree = renderer.create(<Film
    key={0}
    title={`test`}
    onClick={ jest.fn() }
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
