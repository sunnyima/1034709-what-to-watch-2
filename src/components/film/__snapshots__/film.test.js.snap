// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film correctly renders after relaunch 1`] = `
<div>
  <p>
    test
  </p>
  <button
    onClick={[MockFunction]}
  >
    View Info
  </button>
</div>
`;
